docs/website/.gitignore

```
@@ -0,0 +1,26 @@
+# Dependencies
+/node_modules
```

```suggestion
/node_modules
package-lock.json
```

This prevents a certain honking goose from making stupid pushes from their npm based docs builds... 🙃

---

😏